Makefile

```diff
@@ -4,8 +4,6 @@ GO_VERSION := 1.23.7
 LLAMA_SERVER_VERSION := v0.0.4
```

Comment:
nit: should be `latest`?

Reply:
Good point, thanks!
Fixed in https://github.com/docker/model-runner/pull/33/commits/f826ff1830b0830de0b9fb6b93eb55d89c13e0a0.